pkg/ddevapp/traefik_config_template.yaml

```diff
@@ -16,9 +16,8 @@ http:
       rule: Host({{ range $i, $h := $s.ExternalHostnames }}{{if $i}}, {{end}}`{{$h}}`{{end}})
       {{ end }}
       service: "{{$appname}}-{{$s.Service.InternalServiceName}}-{{$s.Service.InternalServicePort}}"
+      middlewares: []
```

Comment:
I don't seem to be able to comment above, but the warnings on #ddev-generated at top of file should point people to the *correct* place to make changes. The comment remains correct, of course, but since there's a *good* way to do this now, point them to it.